locale/fr_FR/submission.xml

```diff
@@ -334,7 +334,7 @@
 	<message key="notification.type.editorDecisionTitle">Dernière décision du rédacteur en chef.</message>
 	<message key="notification.type.editorDecisionAccept">Soumission acceptée.</message>
 	<message key="notification.type.editorDecisionExternalReview">Le processus d'évaluation externe a débuté.</message>
-	<message key="notification.type.editorDecisionPendingRevisions">Demande d'évaluation.</message>
```

Comment:
Could you move these changes to a separate pull request? They aren't related to the issue this PR is associated with.

Reply:
@asmecher : why do you think this is not related to this PR ?

Reply:
think because this changes already existing traslations and we would need an translator (manager) to review that... but, if I am wrong, Alec can say...

Reply:
Hmmm... I saw one more comment in my e-mail notifications but now I cannot see it here :-P @forgive38, yes, those translation changes (especially without changing the en_US texts) should go into a separate PR -- that means that the translation is not quite right or should be improved or ... , right? This would be a topic/PR for it self -- translation improvement... -- and we would need to consult the translator who is responsible for and maintaining that translation to take a look...

Reply:
ok I see
I will do a PR to locale update